.github/workflows/build_and_test_release_latest.yml

```diff
@@ -0,0 +1,52 @@
+name: Build & Test release-latest
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+  schedule:
+    # Run every hour. This helps detect flakiness,
+    # and broken external dependencies.
+    - cron:  '0 * * * *'
+
+jobs:
+  build_and_test_release_latest:
+    name: Build and Test Release Latest ROS ${{ matrix.ros_version }} ${{ matrix.ros_distro }}
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        ros_distro: [kinetic, melodic]
+        include:
+        - ros_distro: kinetic
+          ubuntu_distro: xenial
+          ros_version: 1
+        - ros_distro: melodic
+          ubuntu_distro: bionic
+          ros_version: 1
+    container:
+      image: rostooling/setup-ros-docker:ubuntu-${{ matrix.ubuntu_distro }}-ros-${{ matrix.ros_distro }}-ros-base-latest
+    env:
+      # Needed for the CMakeLists.txt setup
+      ROS_DISTRO: ${{ matrix.ros_distro }}
+      ROS_VERSION: ${{ matrix.ros_version }}
+    steps:
+    # TODO(setup-ros-docker#7): calling chown is necessary for now
+    - run: sudo chown -R rosbuild:rosbuild "$HOME" .
+    # Needed to access the vcs repos file from the workspace
+    - name: Checkout source
+      uses: actions/checkout@v2
+    - uses: ros-tooling/action-ros-ci@0.0.17
```

Comment:
FYI, there is a newer version https://github.com/ros-tooling/action-ros-ci/tags. Perhaps, it might be easier/better to do it as a whole for all migrated repos once these PR sets of yours are approved and merged.

Reply:
I'll address in another PR.